data/frame_meta.go

```diff
@@ -13,16 +13,22 @@ type FrameMeta struct {
 	// Datasource specific values
 	Custom interface{} `json:"custom,omitempty"`
 
-	// Stats is TODO
-	Stats interface{} `json:"stats,omitempty"`
-
-	// This is the raw query sent to the underlying system.  All macros and templating
-	// as been applied.  When metadata contains this value, it will be shown in the query inspector
-	ExecutedQueryString string `json:"executedQueryString,omitempty"`
+	// Stats is an array of query result statistics.
+	Stats []QueryResultMetaStat `json:"stats,omitempty"`
 
 	// Notices provide additional information about the data in the Frame that
 	// Grafana can display to the user in the user interface.
 	Notices []Notice `json:"notices,omitempty"`
+
+	// Transformations is used to track transformation IDs that were part of the processing.
```

Comment:
Is this the Transformations feature that was added in 7.0? (beta, not in explore as far as I know). Or another type of Transformation?

Reply:
@kaydelaney -- do you need this in go now?  On the frontend, this keeps a list of ever transformation (frontend) that ran, and is not really exposed anywhere yet.

Reply:
I have no need for it but added it for parity. I'll remove it if it's not necessary though.